lib/nexmo_developer/spec/services/orbit_feedback_notifier_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe OrbitFeedbackNotifier do
+  let(:workspace_id) { 'abc123' }
+  let(:uri) { "https://app.orbit.love/api/v1/#{workspace_id}/activities" }
+  let(:config_file) { "#{Rails.configuration.docs_base_path}/config/feedback.yml" }
+  let!(:config) { Feedback::Config.find_or_create_config(YAML.safe_load(File.read(config_file))) }
+  let!(:feedback) do
+    Feedback::Feedback.new(
+      config: config,
+      sentiment: 'positive',
+      path: 0,
+      steps: ['nothing!']
+    )
+  end
+
+  subject { described_class.new(feedback) }
+
+  before do
+    allow(ENV).to receive(:[]).and_call_original
+    allow(ENV).to receive(:[]).with('ORBIT_API_KEY').and_return('api_key')
+  end
+
+  describe '#post!' do
+    it 'sends the new feedback to Orbit' do
+      stub_request(:post, uri)
+        .with(headers: { 'Authorization' => "Bearer #{ENV['ORBIT_API_KEY']}", 'Content-Type' => 'application/json' })
+        .to_return(status: 201)
```

Comment:
you are missing the method invocation here `subject.post!`

Reply:
👍🏻